webpack.mix.js

```diff
@@ -57,6 +57,7 @@ mix.setPublicPath('assets/dist')
     .ts('src/Promotions/InPluginUpsells/resources/js/donation-options.ts', 'js/donation-options.js')
     .ts('src/Promotions/InPluginUpsells/resources/js/payment-gateway.ts', 'js/payment-gateway.js')
     .ts('src/Promotions/WelcomeBanner/resources/js/index.tsx', 'js/welcome-banner.js')
+   .ts('src/FormTaxonomies/resources/form-builder/index.tsx', 'js/form-taxonomy-settings.js')
```

Comment:
We use WordPress scripts for the form builder.  Technically this could be done with laravel mix but it's more complicated and we want to move away from this.  So, move this to `wordpress-scripts-webpack.config.js` like this:

```
formTaxonomySettings: srcPath('FormTaxonomies/resources/form-builder/index.tsx'),
```

Reply:
Resolved, see [e43fd0d](https://github.com/impress-org/givewp/pull/7463/commits/e43fd0de9790699d982ed8337e15b7cece75364e).